FlowCal/excel_ui.py

```diff
@@ -170,13 +172,27 @@ def write_workbook(filename, table_list, column_width=None):
     # Pandas' default header format is bold text with thin borders. Here we
     # use bold text only, without borders.
     # The format module is in pd.core.format in pandas<=0.18.0,
-    # pd.formats.format in pandas>=0.18.1.
+    # pd.formats.format in pandas>=0.18.1, and pd.io.formats.excel in
+    # pandas>=0.20.
```

Comment:
I think your intervals are overlapping in this description (e.g. v0.20.2 is both >= 0.18.1 and >= 0.20), but that's really not very important. The code appears to handle it correctly? (having not actually tested it myself)